[2025-07-15T19:41:16.713+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_failure_sensor.delay_task manual__2025-07-15T19:41:12.412193+00:00 [queued]>
[2025-07-15T19:41:16.717+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_failure_sensor.delay_task manual__2025-07-15T19:41:12.412193+00:00 [queued]>
[2025-07-15T19:41:16.717+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-07-15T19:41:16.724+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): delay_task> on 2025-07-15 19:41:12.412193+00:00
[2025-07-15T19:41:16.725+0000] {standard_task_runner.py:57} INFO - Started process 3791 to run task
[2025-07-15T19:41:16.727+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl_failure_sensor', 'delay_task', 'manual__2025-07-15T19:41:12.412193+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/failure_sensor_dag.py', '--cfg-path', '/tmp/tmpq580gp_b']
[2025-07-15T19:41:16.728+0000] {standard_task_runner.py:85} INFO - Job 31: Subtask delay_task
[2025-07-15T19:41:16.746+0000] {task_command.py:410} INFO - Running <TaskInstance: etl_failure_sensor.delay_task manual__2025-07-15T19:41:12.412193+00:00 [running]> on host 1e79179934e4
[2025-07-15T19:41:16.781+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_failure_sensor' AIRFLOW_CTX_TASK_ID='delay_task' AIRFLOW_CTX_EXECUTION_DATE='2025-07-15T19:41:12.412193+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-15T19:41:12.412193+00:00'
[2025-07-15T19:41:16.782+0000] {logging_mixin.py:150} INFO - Delaying for 1 minute...
[2025-07-15T19:42:16.831+0000] {logging_mixin.py:150} INFO - Delay complete.
[2025-07-15T19:42:16.832+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-07-15T19:42:16.840+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=etl_failure_sensor, task_id=delay_task, execution_date=20250715T194112, start_date=20250715T194116, end_date=20250715T194216
[2025-07-15T19:42:16.873+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-07-15T19:42:16.884+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
